actix-cache-derive/src/cacheable_macro.rs

```diff
@@ -41,7 +41,13 @@ pub fn impl_cacheable_macro(ast: &syn::DeriveInput) -> TokenStream {
     let gen = quote! {
         impl Cacheable for #name {
             fn cache_key(&self) -> Result<String, actix_cache::CacheError> {
+                let message_path = std::any::type_name::<#name>()
```

Comment:
@singulared 
About `type_name`. Is it a good idea to use this function? From docs: `In addition, the output may change between versions of the compiler.` https://doc.rust-lang.org/std/any/fn.type_name.html#note
We are using a macro and I think we could find a way to insert the type name into the string without type_name.

Reply:
Done with macro.